### DISCUSSION TITLE
Sentence Embeddings bias

### DISCUSSION THREAD

**Discussion by rafaljanwojcik** [JohnSnowLabs/langtest#462] [Ideas]
Hello, 
do you have in plans to add test cases for Sentence Embeddings models? LLM's can be quite useful in search/recommendation engines, having an option to test them for bias towards i.e., gender might be useful
cheers!

**Comment by ArshaanNazir** [comment]
Hi,
Thank you very much for your valuable suggestion. We appreciate your initiative and believe that it will indeed enhance our system. However, due to current obligations and existing projects in our pipeline, we may not be able to incorporate your suggestion immediately. We assure you, though, that your proposal will be given due consideration in near future.
